test/jasmine/tests/contour_test.js

```diff
@@ -6,7 +6,7 @@ var Lib = require('@src/lib');
 
 var Contour = require('@src/traces/contour');
 var makeColorMap = require('@src/traces/contour/make_color_map');
-var colorScales = require('@src/components/colorscale/scales');
+var colorScales = require('@src/components/colorscale/scales').scales;
```

Comment:
@etpinard Would you mind renaming this variable to `colorScale`? There are only few other places where `colorScales` is used instead of `colorScale`. Namely in {`traces/scatter3d`, `traces/scattergl`, `traces/scatterpolargl` & `traces/splom`}.

Reply:
Cleaned up in https://github.com/plotly/plotly.js/pull/3341/commits/7c21eab5b2bbe6a5529ef91ca3309ef866851cd0 

Thanks for spotting that!